release-notes/SupportedFeatures-v1.0.0.md

```diff
@@ -19,7 +19,7 @@ Supported features
 |Transports    |Http                   |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |Https                  |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:           |
 |              |Tcp                    |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:          |
-|              |WebSockets             |[:warning:](#known_issues)           |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
```

Comment:
I cannot add a comment there... can we remove line 8, 10 and 11 (the tip and :grey_question:) as they are not much useful or not used anymore?